doc/contributing/hw/comportability/README.md

```diff
@@ -462,62 +463,140 @@ Here is an example specification:
+A status interrupt request means that "the condition is currently true".
+
+#### Event Type Interrupt
+
+Event type interrupts are latched indications of defined peripheral events that have occurred and have not yet been acknowledged by the processor.
+For instance, the GPIO module might detect the rising or falling edge of one its inputs as an interrupt event.
+Using level-triggered wired interrupts, this is signalled by latching `intr_o` high (request) when the event occurs, and resetting `intr_o` low (ack) when software has finished executing its handler.
+
+> The generic component `prim_intr_hw` can generate the correct hardware when parameterized with `.IntrT("Event")`.
+
+![Event Type Interrupt Hardware](event_intr_type.svg)
+
+The event is latched into the corresponding bitfield of `INTR_STATE` when it occurs.
+The event is cleared/acknowledged by a sw write of this bitfield, as `INTR_STATE` has **W1C** behaviour in this mode.
+The waveform below shows the timing of the event occurrence, its latched value, and the clearing of the event by a SW handler.
 
 ```wavejson
 {
   signal: [
     { name: 'Clock',             wave: 'p.............' },
-    { name: 'event',             wave: '0..10.........' },
+    { name: 'hw_i',              wave: '0..10.........' },
     { name: 'INTR_ENABLE',       wave: '1.............' },
     { name: 'INTR_STATE',        wave: '0...1....0....' },
-    { name: 'intr_o',            wave: '0...1....0....' },
-    { name: 'SW write to clear', wave: '0.......10....' },
+    { name: 'intr_o',            wave: '0....1....0...' },
+    { name: 'SW w1c',            wave: '0.......10....' },
   ],
   head: {
-    text: 'Interrupt Latching and Clearing',
+    text: 'Event-Type Interrupt Latching and Clearing (output flop enabled)',
   },
   foot: {
-    text: 'event signaled at cycle 3, state bit cleared in cycle 8',
+    text: 'event signaled at cycle 3, cleared in cycle 8',
     tock: 0
   },
 }
 ```
 
-### Status Type Interrupt
+#### Status Type Interrupt
+
+Status type interrupts create a wired level-triggered interrupt signal directly from the input signal from hardware (only qualified by `INTR_ENABLE`).
+If the input signal is asserted, then the wired interrupt is constantly asserted, effectively issuing interrupt requests continuously.
+With this interrupt type, the acknowledgement part of the req/ack handshake is an intervention from sw which causes the input signal from hardware to return to an inactive state.
+This means that the wired interrupt does not deassert until the root cause of the hardware input signal is addressed.
+Software cannot acknowledge this interrupt request using the `INTR_STATE` register, which has (`ro`) acccess permissions in this context.
+If SW cannot address the root cause of the interupt, or wishes to defer handling, it should mask the interrupt, either at the peripheral by setting `INTR_ENABLE[i] = 1'b0`, or at some point further up the tree such as at the PLIC.
```

Comment:
I wonder whether it would be worth moving this last sentence to below the waveform diagram of addressing the cause. Then, after that sentence, include an illustration of the action of disabling the interrupt?
Both of the present diagrams do not show the interaction of INTR_ENABLE, but rather leave it high throughout.

Reply:
Yep, moved the final sentence down below the diagram.
A new illustration would be nice, I will leave that as a follow-up to this PR.